amethyst_gltf/Cargo.toml

```diff
@@ -27,7 +27,7 @@ gltf = "0.14"
 hibitset = { version = "0.6.2", features = ["parallel"] }
 itertools = "0.8"
 log = "0.4.6"
-mikktspace = { version = "0.1" }
+mikktspace = { git = "https://github.com/gltf-rs/mikktspace", rev = "abbae9b486f280ce6e331821e2454c2cfd746d5b" }
```

Comment:
I don't think we should be adding git dependencies to the engine, even if pinned to a revision

Reply:
I agree that it is not the prettiest solution ever, but my point of view is this:

- it fixes an error (am I really the only one who had it?)
- it gets rid of a C dependency
- the `rev` part ensures there are no moving parts, unless we decide to update explicitly
- it works

Is there a practical reason for not doing it this way?

Reply:
- You can't release a crate with git dependency on crates.io
- It's way less flexible than a semver (a fixed revision vs allowed range of versions)

Reply:
> You can't release a crate with git dependency on crates.io

Oh, I did not know that, sorry! In that case, I agree it would not be a good idea.